src/runtime_src/core/pcie/emulation/hw_em/alveo_shim/shim.cxx

```diff
@@ -1092,11 +1094,18 @@ namespace xclhwemhal2 {
 
     std::string simulationDirectoryMsg = "INFO: [HW-EMU 05] Path of the simulation directory : " + getSimPath();
     logMessage(simulationDirectoryMsg);
-
-    sock = std::make_shared<unix_socket>();
+     
+    try{
+       sock = std::make_shared<unix_socket>();
+    }
+    catch(std::exception &e){
```

Comment:
```suggestion
    catch(const std::exception) {
```

Reply:
Now we are using variable e to show the error message.